charts/kong/templates/hpa.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.autoscaling.enabled }}
+apiVersion: {{ .Capabilities.APIVersions.Has "autoscaling/v2beta2" | ternary "autoscaling/v2beta2" "autoscaling/v1" }}
+kind: HorizontalPodAutoscaler
+metadata:
+  name: "{{ template "kong.fullname" . }}"
+  labels:
+    app: "{{ template "kong.name" . }}"
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+    component: app
+spec:
+  scaleTargetRef:
+    apiVersion: extensions/v1beta1
+    kind: Deployment
+    name: "{{ template "kong.fullname" . }}"
+  minReplicas: {{ .Values.autoscaling.minReplicas }}
+  maxReplicas: {{ .Values.autoscaling.maxReplicas }}
+  {{- if not (.Capabilities.APIVersions.Has "autoscaling/v2beta2") }}
+  CPUTargetUtilization:
+    TargetPercentage: {{ .Values.autoscaling.cpuTargetPercentage | default 80 }}
```

Comment:
Is this correct?
The two lines should rather just be:
```
targetCPUUtilizationPercentage: {{}}
```


Reference: https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/

Reply:
Interesting... My bad, TBH, only really used it with `autoscaling/v2beta2` and was just looking at the following doc -> https://github.com/kubernetes/community/blob/master/contributors/design-proposals/autoscaling/horizontal-pod-autoscaler.md#horizontalpodautoscaler-object

But that doc is old. I'll update the PR